iai_apartment/urdf/kitchen.xarco

```diff
@@ -1,38 +1,35 @@
-<?xml version="1.0"?>
-
-<!-- Such is life in a virtual world -->
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="kitchen">
 
-	<!-- Such is life in a kitchen : xacro -->
+	
 	<xacro:macro name="kitchen_macro" params="name parent *origin"> 
 
 		<joint name="${name}" type="fixed">
 			<xacro:insert_block name="origin" />
 			<parent link="${parent}" />
-			<child link="kitchen_root"/>
+			<child link="kitchen_root" />
 		</joint>
 		<link name="kitchen_root" />
 
-<!-- Frames, reference frames -->
-	<joint name="frame_A_joint" type="fixed">
-		<parent link="kitchen_root" />
-		<origin xyz="0.5 0.5 0.080" />
-		<child link="side_A"/>
-	</joint>
 
-	<!-- Bond, James Bond -->
-	<joint name="frame_B_joint" type="fixed">
-		<parent link="kitchen_root" />
-		<origin xyz="3 3.5 0.092" rpy="0 0 3.14"/>
-		<child link="side_B"/>
-	</joint>
+		<joint name="frame_A_joint" type="fixed">
+			<parent link="kitchen_root" />
+			<origin xyz="0.5 0.5 0.080" />
+			<child link="side_A" />
+		</joint>
+
+		
+		<joint name="frame_B_joint" type="fixed">
+			<parent link="kitchen_root" />
+			<origin xyz="3 3.5 0.092" rpy="0 0 3.14" />
+			<child link="side_B" />
+		</joint>
+
+		<link name="side_A" />
 
-	<link name="side_A" />
+		<link name="side_B" /> 
 
-	<link name="side_B" /> 
 
-<!-- Cabinets side A -->
```

Comment:
Hmm, I don't mind removing Bond James Bond comments (although Lara might get upset :( ) but the cabinet comments might be useful. I guess this was to be expected with a script... I wonder if we can make these comments into a proper tag so it wouldn't disappear.